docs/UserGuide.md

```diff
@@ -47,19 +47,19 @@ App-Ointment is a desktop app for for managing and scheduling patient appointmen
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
+* Parameters in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
+* Parameters with `…`​ after them can be used multiple times (including zero times).<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g. if the command format specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```

Comment:
mildly related to this, but how do we specify that the `--force` parameter has to be before the index but after the delete/clear? (ie cannot be in any order)

Reply:
I guess this is a problem since we mentioned that:
"Parameters can be in any order.
e.g. if the command format specifies n/NAME p/PHONE_NUMBER, p/PHONE_NUMBER n/NAME is also acceptable."
Would it be better if we say parameters with prefixes e.g. `n/` can be in any order to avoid confusion?

Reply:
I will also add in that `--force` before the `INDEX` in the sentence for greater clarity.

Reply:
> Would it be better if we say parameters with prefixes e.g. n/ can be in any order to avoid confusion?

this sounds good